### PR TITLE
Updated colors of Slope Plots

### PR DESCRIPTION
**Pull request by BhaswatiRoy**
ref: https://github.com/bokeh/bokeh/issues/11481

1. Improved color accessibility by replacing primary colors with "Sunset" Color Palette with a contrast ratio of 7.18 : 1
2. Increased marker size from 6 to 8.
3. Added a border color Black to make the scatter markers noticeable to users.

image

Preview: 

image

### CODE REVIEW

**Comment by BhaswatiRoy** [comment]
Some more color options I explored while going through the accessible color palettes section of the docs -

1. Crimson + Light Yellow (Sunset)
2. Maroon + Light Blue (BuRd)

image

image

**Comment by bryevdv** [comment]
Same comment as https://github.com/bokeh/bokeh/pull/12999#issuecomment-1479893677 These changes specify named SVG colors. I would expect to use the exact colors from the accessible palettes in `bokeh.palettes`

**Comment by BhaswatiRoy** [comment]
> Same comment as [#12999 (comment)](https://github.com/bokeh/bokeh/pull/12999#issuecomment-1479893677) These changes specify named SVG colors. I would expect to use the exact colors from the accessible palettes in `bokeh.palettes`

Submitted the necessary changes

File: examples/basic/annotations/slope.py [hunk 1]
```diff
@@ -10,21 +10,24 @@
...
+from bokeh.palettes import Sunset10
 
 # linear equation parameters
 slope, intercept = 2, 10
 
 xpts = np.arange(0, 20, 0.2)
 ypts = slope * xpts + intercept + np.random.normal(0, 4, 100)
 
-p = figure(width=450, height=450, x_axis_label='x', y_axis_label='y',
+blueshade, yellowshade = Sunset10[0], Sunset10[5]
```
Comment (bryevdv) [inline examples/basic/annotations/slope.py]: Same as the other PR, let's use names `blue` and `yellow` here
```suggestion
blue, yellow = Sunset10[0], Sunset10[5]
```

**Review by bryevdv — COMMENTED** [review]
also very close, just one comment!

**Comment by BhaswatiRoy** [comment]
> also very close, just one comment!

done!

File: examples/basic/annotations/slope.py [hunk 2]
```diff
@@ -10,21 +10,24 @@
 
 from bokeh.models import Slope
 from bokeh.plotting import figure, show
+from bokeh.palettes import Sunset10
```
Comment (bryevdv) [inline examples/basic/annotations/slope.py]: imports are out of order:
```suggestion
from bokeh.palettes import Sunset10
from bokeh.plotting import figure, show
```

**Comment by BhaswatiRoy** [comment]
resolved!

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @BhaswatiRoy I will merge after CI finishes!

**Comment by BhaswatiRoy** [comment]
> Thanks @BhaswatiRoy I will merge after CI finishes!

thanks for the review

**Pull request merged**